jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+
+**hello-orb**
+```yaml
+# partial yaml from hello-orb
+jobs:
+  sayhello:
+    parameters:
+      saywhat:
+        description: "To whom shall we say hello?"
+        default: "World"
+        type: string
+    machine: true
+    steps:
+      - say:
+          saywhat: "<< parameters.saywhat >>"
+commands:
+  saywhat:
+    parameters:
+      saywhat:
+        type: string
+    steps:
+      - run: echo "<< parameters.saywhat >>"
+```
+
+**Config leveraging hello-orb**
+```yaml
+# config.yml
+version: 2.1
+orbs:
+  hello-orb: somenamespace/hello-orb@volatile
+workflows:
+  build:
+    jobs:
+      - hello-orb/sayhello:
+          saywhat: Everyone
+```
+--->
+
+### Parameter Scope
+{:.no_toc}
+
+Parameters are in-scope only within the job or command that defined them. If you want a job or command to pass its parameters to a command it invokes, they must be passed explicitly. Command, job, executor, and parameter names can only contain lowercase letters a-z, digits, and _ and -, and must start with a letter.
+
+
+```yaml
+version: 2.1
+
+jobs:
+  sayhello:
+    parameters:
+      saywhat:
+        description: "To whom shall we say hello?"
+        default: "World"
+        type: string
+    machine: true
+    steps:
+      - say:
+          # Since the command "say" doesn't define a default
+          # value for the "saywhat" parameter, it must be
+          # passed in manually
+          saywhat: << parameters.saywhat >>
+
+commands:
+  say:
+    parameters:
+      saywhat:
+        type: string
+    steps:
+      - run: echo "<< parameters.saywhat >>"
+
+workflows:
+  build:
+    jobs:
+      - sayhello:
+          saywhat: Everyone
+```
+
+
+
+### Invoking the Same Job Multiple Times
+{:.no_toc}
+
+A single configuration may invoke a job many times. At configuration processing time during build ingestion, CircleCI will auto-generate names if none are provided.  If you care about the name of the duplicate jobs, they can be explicitly named with the `name` key.
+
+**Note:** The user must explicitly name repeat jobs when a repeat job should be upstream of another job in a workflow (ie: if the job is used under the `requires` key of a job invocation in a workflow you will need to name it).
+
+```yaml
+version: 2.1
+workflows:
+  build:
+    jobs:
+      - loadsay
+      # This doesn't need an explicit name as it has no downstream dependencies
+      - sayhello:
+          saywhat: Everyone
+          requires:
+            - loadsay
+      # This needs an explicit name for saygoodbye to require it as a job dependency
+      - sayhello:
+          name: SayHelloChad
+          saywhat: Chad
+      # Uses explicitly defined "sayhello"
+      - saygoodbye:
+          requires:
+            - SayHelloChad
+```
+
+### Pre and Post Steps
+{:.no_toc}
+
+**Note:** The keys `pre-steps` and `post-steps` in jobs are available in configuration version 2.1 and later_
```

Comment:
The extra "_" at the end of this line may be a typo

Reply:
fixed, thanks.